src/components/Amendment/Amendment.js

```diff
@@ -201,6 +204,7 @@ export default class Amendment extends React.Component {
                 </Col>
               </Row>
               <AccordionSet initialStatus={this.getInitialAccordionsState()}>
+                {license?.amendments?.length > 1 && <LicenseAmendments {...this.getSectionProps('licenseAmendments')} licenseAmendmentsAccordionLabel={this.licenseAmendmentsAccordionLabel} />}
```

Comment:
You can just pass the label defined on L99 inline instead of defining a new const:
```suggestion
                {license?.amendments?.length > 1 && 
                <LicenseAmendments {...this.getSectionProps('licenseAmendments')} 
                licenseAmendmentsAccordionLabel={<FormattedMessage id="ui-licenses.section.amendmentsOnParentLicense" />} />}
```